egs2/sinhala/asr1/local/score.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright 2021  Karthik Ganesan 
```

Comment:
@siddhu001 seems to have a unified scoring function, can you try that?

Reply:
Sure , added symlink to the unified scoring file

Reply:
Can you also remove the file?

Reply:
Yes sure updated